packages/p2p/src/components/app.jsx

```diff
@@ -379,6 +383,7 @@ class App extends React.Component {
                     order_table_type,
                     changeOrderToggle: this.changeOrderToggle,
                     createAdvertiser: this.createAdvertiser.bind(this),
+                    resetNicknameErrorState: this.resetNicknameErrorState.bind(this),
```

Comment:
`.bind(this)` is not needed since it's an arrow func

Reply:
Ahh alright, I'll make this change, thank you Bala! 🙏